src/lib/constants.ts

```diff
@@ -50,6 +50,11 @@ const constants = {
 	bootBlockDevice: '/dev/mmcblk0p1',
 	hostConfigVarPrefix: 'HOST_',
 	migrationBackupFile: 'backup.tgz',
+	// Use this failure multiplied by 2**Number of failures to increase
+	// the backoff on subsequent failures
+	backoffIncrement: 500,
+	// The maximum time to backoff on repeated failure
+	maxBackoffTime: 30000,
```

Comment:
While we're at it, we might wanna consider increasing the max backoff time here? In the APIBinder [we backoff all the way up to the polling interval](https://github.com/balena-io/balena-supervisor/blob/master/src/api-binder.ts#L528) (which by default is 10 minutes now), and I wonder if here we should also do the same, to reduce bandwidth usage for devices that are failing to download for whatever reason, and also further reduce pressure on the registry/delta server/API.